core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -289,13 +289,11 @@ export class HorizontalBarChart extends Chart {
     });
   }
 
-  _displayBars(barGroups, enteringBarGroups) {
+  _displayBars(enteringBarGroups) {
     this._addEnteringBars(enteringBarGroups);
-    this._updateExistingBars(barGroups);
   }
 
-  _displayValues(barGroups, enteringBarGroups) {
-    this._updateExistingBarValues(barGroups);
+  _displayValues(enteringBarGroups) {
```

Comment:
You could go the whole nine yards and remove those functions...

Reply:
was thinking they could be useful later on, the updating functions aren't really the problem here, it is the rerendering, so maybe in future use they could actually be functional

Reply:
We can always bring them back; if they're not needed _right now_, let's just eliminate them right now.